web/api/v2/v2pb/api.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto3";
+package prometheus.api.v2;
+
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "v2pb";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_getters_all) = false;
+
+// AdminTSDB exposes methods of Prometheus' TSDB.
+service AdminTSDB {
+  // Reload triggers a reload of block data.
+  rpc Reload(AdminTSDBReloadRequest) returns (AdminTSDBReloadResponse) {
+    option (google.api.http) = {
+      get: "/admin/tsdb/reload"
```

Comment:
Yes, hence the handler says "not implemented". We can extend the API incrementally once we've settled on the basics here.